suite-native/atoms/src/Link.tsx

```diff
@@ -0,0 +1,13 @@
+import React, { useCallback } from 'react';
+import {Linking, TouchableOpacity, TouchableOpacityProps} from 'react-native';
+
+type LinkProps = TouchableOpacityProps & {
+    url: string;
+};
+
+export const Link = ({ url, ...touchableProps }: LinkProps) => {
+    const handleRedirect = useCallback(async () => {
```

Comment:
I think this is one of places where useCallback is redundant. `TouchableOpacity` where it is passed is not memoized be default so it will rerender everytime when parent component is rerender anyway. Please remove it then.

Reply:
fixed